indico/modules/events/contributions/forms.py

```diff
@@ -32,3 +33,16 @@ class ContributionForm(IndicoForm):
     duration = TimeDeltaField(_("Duration"), [DataRequired()], default=timedelta(minutes=20),
                               units=('minutes', 'hours'),
                               description=_("The duration of the contribution"))
+
+
+class ContributionProtectionForm(IndicoForm):
+    managers = PrincipalListField(_('Managers'), description=_('List of users allowed to modify the contribution'),
+                                  serializable=False, groups=True)
+    protection_mode = IndicoRadioField(_('Protection mode'), orientation='horizontal',
```

Comment:
Why not `IndicoEnumSelectField` (you can use the radio widget anyway, maybe add a new field class to avoid specifying the rather ugly jinjawidget call here)